docs/typography/typography.html

```diff
@@ -0,0 +1,320 @@
+                        <i class="fas fa-2x fa-times"
+                            aria-hidden="true"></i>
+                    </button>
+                </div>
+            </div>
+        </header>
+        <!-- main content -->
+        <div class="main-wrapper">
+            <aside class="main__sidebar px-9 py-9">
+                <ul class="main__sidebar--list">
+                    <li class="display-none mobile-menu-btn display-flex"> Menu <i
+                            class="fas fa-angle-double-right arrow-icon"></i>
+                    </li>
+                    <li class="sidebar--chapter"><a href=""></a>Get Started</li>
+                    <li><a href="../getstarted.html#introduction">Introduction</a></li>
+                    <li><a href="../getstarted.html#installation">Installation</a></li>
+                    <li>
+                        <a href="../getstarted.html#starter-template">Starter Template</a>
+                    </li>
+                    <li class="sidebar--chapter"><a href=""></a>Components</li>
+                    <li><a href="../alert/alert.html">Alert</a></li>
+                    <li>
+                        <a href="../avatar/avatar.html">Avatar</a>
+                    </li>
+                    <li><a href="../badge/badge.html">Badge</a></li>
+                    <li><a href="../button/button.html">Button</a></li>
+                    <li><a href="../card/card.html">Card</a></li>
+                    <li><a href="../image/image.html">Image</a></li>
+                    <li><a href="../input/input.html">Input</a></li>
+                    <li><a href="../navigation/navigation.html">Navigation</a></li>
+                    <li>
+                        <a class="active-link"
+                            href="../typography/typography.html">Typography</a>
+                    </li>
+                    <li><a href="../snackbar/snackbar.html">Snackbar</a></li>
+                </ul>
+            </aside>
+            <section class="main__section">
+                <h1 class="h1">Typography</h1>
+                <hr />
+                <section id="typography-introduction">
+                    <h2>Introduction</h2>
+                    <p> Typography is an integral part of any component library. ClutchUI provides various typography
+                        class aswell as some text utilities. </p>
+                </section>
+                <!-- Heading -->
+                <h2>Heading</h2>
+                <section id="typography-heading">
+                    <p>All heading tags from <code>&lt;h1&gt; to &lt;h6&gt; </code></p>
+                    <div class="demo-container-typography mt-12">
+                        <h1>Heading h1</h1>
+                        <h2>Heading h2</h2>
+                        <h3>Heading h3</h3>
+                        <h4>Heading h4</h4>
+                        <h5>Heading h5</h5>
+                        <h6>Heading h6</h6>

```

Comment:
Instead consider trying to apply your own CL styles using utility class, as this can easily be done by the user itself and serves no purpose.

Reply:
Yes, I have implemented them, forgot to add them here. Will update it.